content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -14,7 +14,8 @@ which only handles intra-cluster destinations.
 This task describes how to configure Istio to expose external services to Istio-enabled clients.
 You'll learn how to enable access to external services by defining
```

Comment:
To make the goals of the task clearer, it would be better to use a list:
```suggestion
This task shows three ways to enable access to external services:
```